### DISCUSSION TITLE
Welcome to cub3D Discussions!

### DISCUSSION THREAD

**Discussion by lmangall** [lmangall/Raycaster-Game-Engine#167] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 552020** [comment]
> Ask questions you’re wondering about.

Where are we coming from? Where are we going to? Where are we?

> Share ideas.

💡

> Engage with other community members.

How to engage with the other community member directly, like tagging 'them'? The @ was not working before but now! @lmangall 

> Welcome others and are open-minded. Remember that this is a community we
> build together 💪.

I am as open-minded as a foo-dog can be!

**Comment by lmangall** [comment]
> Where are we coming from? Where are we going to? Where are we?
Github x 3

@552020, That's how you feel when you get "tagged"

Poles are the best foo-dogs, have always been
https://www.youtube.com/watch?v=-jaqg-NMz9A&ab_channel=SaturdayNightLive

**Reply by 552020** [reply]
These filthy greasy meatball-eating, crotch-grabbers Italians, who are not even real white people! If Kars only knew what you watch!

**Reply by lmangall** [reply]
Since we keep making cinema reference (or is it just me ?) 
This is one of the best intro scene I have seen, with a wonderful Italian crotch grabbing move:
https://www.youtube.com/watch?v=HVEqy6K18Yo

**Comment by 552020** [comment]
> > Where are we coming from? Where are we going to? Where are we?
> > Github x 3
> 
> @552020, That's how you feel when you get "tagged"
> 
> Poles are the best foo-dogs, have always been https://www.youtube.com/watch?v=-jaqg-NMz9A&ab_channel=SaturdayNightLive

You can also quote reply here!

**Reply by lmangall** [reply]
The nested conversation UI is pretty well made
